### DISCUSSION TITLE
请问怎么在手机移动端上访问电脑上部署的ChuanhuGPT

### DISCUSSION THREAD

**Discussion by newmou** [GaiZhenbiao/ChuanhuChatGPT#793] [General]
想在手机上使用，不知怎么访问在电脑上部署的ChuanhuGPT

**Comment by wushiyi4** [comment]
手机上要访问，个人觉得有2个思路
1、手机远程连接到电脑上，在电脑上开浏览器访问。
2、部署时允许局域网访问，然后手机接入和电脑相同的局域网，通过手机浏览器，访问部署了chuanhu的电脑   ip:端口。
  2.5   把你的电脑开放到公网上，然后通过公网ip+端口访问

**Comment by GaiZhenbiao** [comment]
在config里配置share参数，可以生成可以外网访问的分享链接。
